WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationSettingDetailsViewController.swift

```diff
@@ -151,42 +151,27 @@ public class NotificationSettingDetailsViewController : UITableViewController
         return cell!
     }
 
-    public override func tableView(tableView: UITableView, heightForHeaderInSection section: Int) -> CGFloat {
-        guard section == firstSectionIndex else {
-            return 0
-        }
-
-        return WPTableViewSectionHeaderFooterView.heightForHeader(siteName, width: view.bounds.width)
-    }
-
-    public override func tableView(tableView: UITableView, viewForHeaderInSection section: Int) -> UIView? {
+    public override func tableView(tableView: UITableView, titleForHeaderInSection section: Int) -> String? {
         guard section == firstSectionIndex else {
             return nil
         }
-
-        let headerView      = WPTableViewSectionHeaderFooterView(reuseIdentifier: nil, style: .Header)
-        headerView.title    = siteName
-        return headerView
+        return siteName
     }
 
-    public override func tableView(tableView: UITableView, heightForFooterInSection section: Int) -> CGFloat {
-        guard let footerText = sections[section].footerText else {
-            return CGFloat.min
-        }
-
-        return WPTableViewSectionHeaderFooterView.heightForFooter(footerText, width: view.bounds.width)
+    public override func tableView(tableView: UITableView, willDisplayHeaderView view: UIView, forSection section: Int) {
+        WPStyleGuide.configureTableViewSectionHeader(view)
     }
 
-    public override func tableView(tableView: UITableView, viewForFooterInSection section: Int) -> UIView? {
+    public override func tableView(tableView: UITableView, titleForFooterInSection section: Int) -> String? {
         guard let footerText = sections[section].footerText else {
```

Comment:
Again, you can probably just return `sections[section].footerText` directly here.